### PR TITLE
HCAP-1051 | HCAP-1030 | Graduation Status | Frontend api integration 

### PR DESCRIPTION
**Pull request by fwpushan**
- [x] Create FE integration layer for post-hire status
 - [x] Show post hire status in Participant detail page

### CODE REVIEW

**Review by dbayly-freshworks — APPROVED** [review]
I feel that `post_secondary_education_underway` and `post_secondary_education_completed` feel a bit wordy, but that's totally a nit. 

Otherwise looks good to me 👍

**Comment by fwpushan** [comment]
> I feel that `post_secondary_education_underway` and `post_secondary_education_completed` feel a bit wordy, but that's totally a nit.
> 
> Otherwise looks good to me 👍

Let discuss and  decide something shorter.
